roles/ipxe/tasks/main.yml

```diff
@@ -48,6 +62,20 @@
     - ipxe.pxe
     - undionly.kpxe
 
+- name: fetch the resulting files efi 64 bit file
+  fetch: src={{ ansible_env.HOME }}/ipxe/src/bin-x86_64-efi/{{ item }}
+        dest=/tmp/on-imagebuilder/ipxe/bin-x86_64-efi{{ item }} fail_on_missing=yes
```

Comment:
Can we set this directory path as a variable instead? Perhaps in https://github.com/RackHD/on-imagebuilder/blob/master/hosts#L21 or in group_vars?